Lib/test/test_inspect.py

```diff
@@ -4155,18 +4155,9 @@ def test(a, *args, b, z=100, **kwargs):
         self.assertEqual(ba.args, (10, 20))
 
     def test_signature_bind_positional_only(self):
-        P = inspect.Parameter
-
-        def test(a_po, b_po, c_po=3, foo=42, *, bar=50, **kwargs):
+        def test(a_po, b_po, c_po=3, /, foo=42, *, bar=50, **kwargs):
```

Comment:
The same here :)

Reply:
Ah yeah, but reluctant to change it now after merging main now that it's no longer in the diff. Is that alright?

Reply:
Yes, I've fixed it in `main` :)